lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -219,7 +219,7 @@ export default function SubmitGradeForm({ student }) {
       </button>
       {!!submitGradeError && (
         <ErrorDialog
-          title="Submit Grade Error"
+          title={`There was a problem submitting ${student ? student.displayName : ' the student'}'s grade.`}
```

Comment:
I could add the `LISOutcomeServiceUrl` but I'm not sure there is a lot of value as I would assume a customer would know what LMS they were using. A screen could show that too.